lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -82,9 +82,11 @@ class OptimizedImages extends Gatherer {
       const isSameOrigin = URL.hostsMatch(pageUrl, record._url);
```

Comment:
hm does this actually taint CORS/require headers? you won't be able to do https -> http and if you're doing http -> https we'll complain elsewhere anyway. I'm cool leaving it since nowhere else compares origins, but I guess we should go on a hunt to find all cases it might matter :)